openshift/backup/backup-deploy.silver.param

```diff
@@ -0,0 +1,39 @@
+#=========================================================
```

Comment:
we are removing the original backup container and switching to helm chart, as a result those oc templates will be gone once I merge in. let's see if we want to turn this into the helm config file later on or just keep the templates for a bit longer.